content/operate/rs/security/access-control/create-roles/create-combined-roles.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: Create roles with combined access
+alwaysopen: false
+categories:
+- docs
+- operate
+- rs
+description: Create roles with both cluster and database access.
+linkTitle: Create roles with combined access
+weight: 70
+---
+
+To create a role that grants database access privileges and allows access to the Cluster Management UI and REST API:
+
+1. [Define Redis ACLs](#define-redis-acls) that determine database access privileges.
+
+1. [Create a role with ACLs](#create-role) added and choose a **Cluster management role** other than **None**.
+
+## Define Redis ACLs
+
+To define a Redis ACL rule that you can assign to a role:
+
+1. From **Access Control > Redis ACLs**, you can either:
+
+    - Point to a Redis ACL and select {{< image filename="/images/rs/buttons/edit-button.png#no-click" alt="The Edit button" width="25px" class="inline" >}} to edit an existing Redis ACL.
+
+    - Select **+ Add Redis ACL** to create a new Redis ACL.
+
+1. Enter a descriptive name for the Redis ACL. This will be used to reference the ACL rule to the role.
```

Comment:
```suggestion
1. Enter a descriptive name for the Redis ACL. This will be used to associate the ACL rule with the role.
```

Reply:
Maybe there is a reason you are using the word 'reference' that I miss, if not it might be more clear to say 'associate with' here.